ports/esp32/boards/SBD_BEES3/board.json

```diff
@@ -0,0 +1,20 @@
+{
+    "deploy": [
+        "../deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "BLE",
+        "WiFi",
+        "USB-C",
+        "RGB LED"
+    ],
+    "images": [
+        "	bees3.jpg"
```

Comment:
Are the leading spaces intended?

Reply:
no they weren't intentional. I've fixed them.